src/state/actions/dispatchArIOInteraction.ts

```diff
@@ -57,6 +60,22 @@ export default async function dispatchArIOInteraction({
             ao: ao,
             scheduler: scheduler,
           });
+          const antRegistry = ANTRegistry.init({
+            signer,
+            processId: ANT_REGISTRY_ID,
+          }) as AoANTRegistryWrite;
+          await antRegistry
+            .register({
+              processId: antProcessId,
+            })
+            .catch((error) => {
+              eventEmitter.emit(
+                'error',
+                new Error(
+                  `Failed to register ANT process: ${error}. You may need to manually register the process`,
```

Comment:
Do we have a way for non-dev users to manually register?

Reply:
Note: this needs to get sorted out in a future PR as we're telling users they have a path to do something but then not really having one.